cybershell/config.py

```
@@ -0,0 +1,153 @@
+    'load_config',  # Legacy function
+    'save_config'   # Legacy function
+]
+
+# Legacy Config class for backward compatibility
+class Config(UnifiedConfig):
+    """
+    Legacy Config class - redirects to UnifiedConfig
+    Deprecated: Use UnifiedConfig instead
+    """
+    
+    def __init__(self, *args, **kwargs):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)